### ISSUE TITLE
Any benchmark?

### ISSUE THREAD

**Issue by leedstyh** [cznic/kv#30]
Great job, and any benchmark vs other pure Golang KV storege like goleveldb (https://github.com/syndtr/goleveldb)?

**Comment by vtolstov** [comment]
@cznic can you provide some numbers?

**Comment by cznic** [comment]
```
~/src/github.com/cznic/kv> git status 
On branch master
Your branch is up-to-date with 'origin/master'.
nothing to commit, working tree clean
~/src/github.com/cznic/kv> git --no-pager log -1
commit 892ccf731fb7aa5e9aa300eb24456d1519afcfc7 (HEAD -> master, origin/master, origin/HEAD)
Author: mockturtl <mockturtl@users.noreply.github.com>
Date:   Mon May 15 16:27:33 2017 -0400

    fix kvaudit (#35)
    
    Fixes #34.
    
    Partial revert of 48b826e.
jnml@4670:~/src/github.com/cznic/kv> go test -bench . -benchmem
goos: linux
goarch: amd64
pkg: github.com/cznic/kv
BenchmarkDelete16-4      	   20000	     87764 ns/op	     461 B/op	      19 allocs/op
BenchmarkExtract16-4     	   20000	     87952 ns/op	     398 B/op	      17 allocs/op
BenchmarkFirst16-4       	  300000	      3727 ns/op	   16727 B/op	       6 allocs/op
BenchmarkGet16-4         	   20000	     82896 ns/op	   31459 B/op	      19 allocs/op
BenchmarkInc-4           	 1000000	      1547 ns/op	      36 B/op	       4 allocs/op
BenchmarkLast16-4        	  500000	      3988 ns/op	   16726 B/op	       6 allocs/op
BenchmarkPut16-4         	   20000	     97242 ns/op	     402 B/op	      14 allocs/op
BenchmarkSet16-4         	   20000	     85449 ns/op	     376 B/op	      14 allocs/op
BenchmarkSeek-4          	   20000	     85874 ns/op	   31527 B/op	      22 allocs/op
BenchmarkNext1e3-4       	   10000	    166837 ns/op	   24000 B/op	    3000 allocs/op
BenchmarkEnumerateDB-4   	   10000	    174696 ns/op	   57059 B/op	    3010 allocs/op
PASS
ok  	github.com/cznic/kv	44.827s
~/src/github.com/cznic/kv> 
```
